### PR TITLE
evolution of compose explainer

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

**Comment by aevesdocker** [comment]
@glours thoughts?  https://deploy-preview-16521--docsdocker.netlify.app/compose/compose-v2/

Let me know if you feel I need to add anything extra, particularly around the differences between V1 and V2. 
I'm keen to strike the right balance between enough information for users to make it clearer, but not so much information that it overwhelms them and makes them less likely to read it all.

File: compose/compose-v2/index.md [hunk 1]
```diff
@@ -1,45 +1,158 @@
...
+The first release of Compose, written in Python, happened at the end of 2014. Between 2014 and 2017, two more versions of Compose were released:
 
-If you rely on using Docker Compose as `docker-compose` (with a hyphen), you can
-set up Compose V2 to act as a drop-in replacement of the previous `docker-compose`.
-Refer to the [Installing Compose](../install/index.md) section for detailed instructions.
+- [Compose 1.6.0 with file format V2](../compose-file/compose-file-v2/)
+- [Compose 1.10.0 with file format V3](../compose-file/compose-file-v3/)
 
-## Context of Docker Compose evolution
+These three early releases are collectively referred to as Compose V1.
```
Comment (glours) [inline compose/compose-v2/index.md:18]: There was way more than 3 releases for Compose v1, the last one is the `v1.29.2` 😅 
But the `1.6.0` and `1.10.0` were noticeable as they introduced version `2` and `3` of the Compose file format.

**Review by glours — APPROVED** [review]
Except the `History` paragraph which isn't clear IHMO, this PR is a great addition 🚀

File: compose/compose-v2/index.md (hunk 1 above)
Comment (aevesdocker) [reply compose/compose-v2/index.md:18]: Ah that's actually what i was trying to communicate in that section - that there were 3 main releases within Compose V1 which are the file formats, not that there were only three releases 😅  But I've obviously not done a good job, so I'll try again! Thank you, Guillaume!

Comment (glours) [reply compose/compose-v2/index.md:18]: Maybe we can reword this way:
```markdown
The first release of Compose, written in Python, happened at the end of 2014. Between 2014 and 2017, two other noticeable versions of Compose, as they introduced new Compose file format version, were released:
```

File: compose/compose-v2/index.md [hunk 2]
```diff
@@ -1,45 +1,159 @@
...
+Between 2014 and 2017 two other noticeable versions of Compose, which introduced new file format versions, were released:
 
-If you rely on using Docker Compose as `docker-compose` (with a hyphen), you can
-set up Compose V2 to act as a drop-in replacement of the previous `docker-compose`.
-Refer to the [Installing Compose](../install/index.md) section for detailed instructions.
+- [Compose 1.6.0 with file format V2](../compose-file/compose-file-v2/)
+- [Compose 1.10.0 with file format V3](../compose-file/compose-file-v3/)
 
-## Context of Docker Compose evolution
+These three key file format versions and releases prior to v1.27.0 are collectively referred to as Compose V1.
```
Comment (glours) [inline compose/compose-v2/index.md:19]:
```suggestion
These three key file format versions and releases prior to v1.29.2 are collectively referred to as Compose V1. 
```
Compose V1 refer to Python version of Compose, even if versions between `v1.27.0` and `v1.29.2` could handle Compose Specification file format they are still considered as Compose V1

**Review by glours — CHANGES_REQUESTED** [review]

**Review by glours — APPROVED** [review]
LGTM

**Review by usha-mandya — APPROVED** [review]
LGTM. Thanks for clarifying the difference in version numbers!

**Review by milas — APPROVED** [review]
Nice improvements! And 💯 for cutting down on number of pages.

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**